doc/lsst.obs.decam/using-obs-decam.rst

```diff
@@ -0,0 +1,79 @@
+.. _lsst.obs.decam.gen3:
+
+#################
+The Gen3 workflow
+#################
+
+These directions are for users who wish to create their own data repositories independent of any Rubin/LSST-supported shared repositories.
+
+Create a Gen3 data repository and write curated calibration products to it (these include crosstalk, linearity, and defects). The latter step creates the `DECam/calib` collection.
+
+.. prompt:: bash
+
+   cd /path/to/desired/repo
+   butler create .
+   butler register-instrument . lsst.obs.decam.DarkEnergyCamera
+   butler write-curated-calibrations . lsst.obs.decam.DarkEnergyCamera
+
+Ingest raw science and calibration (bias and flat) frames, which creates the `DECam/raw/all` collection. Run overscan correction on all the raws so they may be used as crosstalk sources during ISR.
+
+This example assumes the user has setup the `~lsst.cp.pipe`_ module, which is needed for constructing calibration products from raws.
+
+.. prompt:: bash
+
+   butler ingest-raws . /path/to/raw/science/*.fits.fz --transfer link
+   butler ingest-raws . /path/to/raw/calibs/*.fits.fz --transfer link
+   pipetask run -b . -i DECam/raw/all -o DECam/raw/crosstalk-sources -p $CP_PIPE_DIR/pipelines/DarkEnergyCamera/RunIsrForCrosstalkSources.yaml
+
+Build nightly (or similar) bias and flat frames and certify them into a calib collection.
+
+This example assumes the user has a single night of observations with bias frames numbered 1-6 and flat frames numbered 7-12, and wants to create nightly calib products valid for 24 hours. Note the bias building pipeline is camera-agnostic, but the flat-building pipeline has a prerequisite DECam-specific step which correctly handles inter-chip crosstalk.
+
+.. prompt:: bash
+
+   pipetask run -d "exposure IN (1, 2, 3, 4, 5, 6)" -b . -i DECam/raw/all,DECam/calib -o u/username/bias-construction-night1 -p $CP_PIPE_DIR/pipelines/cpBias.yaml --register-dataset-types
+   butler certify-calibrations . u/username/bias-construction-night1 DECam/calib/run1 bias --begin-date 2021-01-01T00:00:00 --end-date 2021-01-01T23:59:59
+   pipetask run -d "exposure IN (7, 8, 9, 10, 11, 12)" -b . -i DECam/raw/all,DECam/raw/crosstalk-sources,DECam/calib -o u/username/flat-construction-night1 -p $CP_PIPE_DIR/pipelines/DarkEnergyCamera/cpFlat.yaml --register-dataset-types
+   butler certify-calibrations . u/username/flat-construction-night1 DECam/calib/run1 flat --begin-date 2021-01-01T00:00:00 --end-date 2021-01-01T23:59:59
+
+Now you can proceed with running ISR and other "processCcd" tasks via a Gen3 pipeline. Some useful pipelines can be found in the `pipelines/DarkEnergyCamera` directory of the `~lsst.ap.pipe`_ package.
+
+
+.. _lsst.obs.decam.gen2:
+
+############################
+The deprecated Gen2 workflow
+############################
+
+Create a Gen2 data repository directory:
+
+.. prompt:: bash
+
+   mkdir /path/to/repo
```

Comment:
For anything with a path like this, it's probably better to have `set BUTLER_PATH=/path/to/repo` as the first step, and then refer to `$BUTLER_PATH` everywhere below, so that the user doesn't actually have to do any search and replace.

Reply:
I don't personally find utilizing environment variables like this very newbie-user-friendly tbh. I saw advice from someone... Tim? K-T? Jim? that working in the butler repo isn't necessarily a good plan, so I will at least change all the dots to REPO. I'm not going to much with the Gen2 stuff.

Reply:
Responding here (even though this is the gen2 section: yes, I used similar comments):

What is the problem with environment variables? It succinctly solves the the copy/paste problem of paths, and makes scripting tasks like this easier. We're not (or, at least I don't think we are) documenting "how to ingest data into the Rubin Pipeline for a complete commandline-user newbie".

Reply:
You _say_ we're not doing that, but I'd wager the majority of folks who bother to click through to these docs will be doing exactly that. Maybe I'm unique in saying I don't like environment variables for tutorials. I feel like they add a level of complexity without adding any clarity. Put yourself in the shoes of an undergrad who is learning unix, python, Rubin shenanigans, FITS files, git, all of it for the first time. It's one more thing when there doesn't need to be one more thing.

Reply:
If they're learning Rubin for the first time, they *really* shouldn't be trying to ingest data, especially right now.